<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherWindow</class>
 <widget class="QMainWindow" name="OtherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeacherWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Students:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="StudentList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="GradeLine"/>
        </item>
        <item>
         <widget class="QPushButton" name="GradeButton">
          <property name="text">
           <string>Grade</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
